docs/topics/api.rst

```diff
@@ -198,6 +188,13 @@ SpiderLoader API
        :param request: queried request
        :type request: :class:`~scrapy.Request` instance
 
+    :meth:`from_settings`:
+
+       This class method is used by Scrapy to create an instance of the class.
+       It's called with the current project settings, and it loads the spiders
+       found recursively in the modules of the :setting:`SPIDER_MODULES`
+       setting.
```

Comment:
I am thinking factory method signatures should still be kept in the API reference. It is specially important because some components may get extra arguments, I believe, and in those cases we should document them in one of the methods (e.g. `from_crawler`) and in the other method refer to the former for parameter details.

What we can do now is simplify the description, both of `from_crawler` and `from_settings`, to something like:

```
Factory method. See :ref:`class-methods`.
```

Assuming we also add `.. _class-methods:` at the beginning of the new topic.

Also, I imagine (please check) that this class also supports `from_crawler`. In that case, we should include an entry for that class method as well, with its own signature, but the same description that basically points to the new topic.